engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -18,46 +20,48 @@ object EngineJobExecutionActor {
 
   /** Commands */
   sealed trait EngineJobExecutionActorCommand
-  case class Execute(jobKey: BackendJobDescriptorKey) extends EngineJobExecutionActorCommand
+  case object Execute extends EngineJobExecutionActorCommand
 
-  case class JobRunning(jobDescriptor: BackendJobDescriptor, backendJobExecutionActor: ActorRef)
+  final case class JobRunning(jobDescriptor: BackendJobDescriptor, backendJobExecutionActor: ActorRef)
```

Comment:
This appears to be a response and not a command (all I did here was add the `final`).